packages/cf-builder-pagination/src/PaginationBuilder.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Pagination, PaginationItem } from 'cf-component-pagination';
-import Icon from 'cf-component-icon';
+import Icon from '../../cf-component-icon/src';
```

Comment:
This is probably wrong since this will break these components. Wondering what the best solution here would be. Probably a webpack/babel alias?